### PR TITLE
Updating Apache Dockerfile to include missing M2 php extensions and d…

### PR DESCRIPTION
**Pull request by chasecbw**
…ependencies

### CODE REVIEW

File: Dockerfile.apache [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM php:7-apache
+FROM php:5.6-apache
```
Comment (bryanlittlefield) [inline Dockerfile.apache]: Change this to the current php version `7.0-apache`

File: Dockerfile.apache [hunk 2]
```diff
@@ -48,6 +48,13 @@ RUN docker-php-ext-install pdo_mysql pdo_pgsql pdo_sqlite
 RUN docker-php-ext-install soap
 RUN docker-php-ext-install tokenizer
 RUN docker-php-ext-install zip
+RUN apt-get install -y zlib1g-dev libicu-dev g++
```
Comment (bryanlittlefield) [inline Dockerfile.apache]: Move this line up above the php config section and into UPDATE/UPGRADE APT PACKAGES section  ( round line: 34

File: Dockerfile.apache [hunk 3]
```diff
@@ -48,6 +48,13 @@ RUN docker-php-ext-install pdo_mysql pdo_pgsql pdo_sqlite
...
+RUN docker-php-ext-configure intl
+RUN docker-php-ext-install intl
+RUN apt-get install -y libxslt-dev
```
Comment (bryanlittlefield) [inline Dockerfile.apache]: Move this line up above the php config section and into `UPDATE/UPGRADE APT PACKAGES` section  ( round line: 34

File: Dockerfile.apache [hunk 4]
```diff
@@ -93,4 +100,4 @@ ADD etc/apache2/vhost.conf /etc/apache2/sites-available/default.conf
 ADD etc/php/php.ini /usr/local/etc/php/
 
 RUN a2enmod rewrite
-RUN a2ensite default
+RUN a2ensite default
```
Comment (bryanlittlefield) [inline Dockerfile.apache]: Was this previously removed?

Comment (bryanlittlefield) [inline Dockerfile.apache]: This looks good. Disregard

**Comment by chasecbw** [comment]
Updated and ready for review!

**Pull request merged**